### ISSUE TITLE
Atom doesn't ignore the files in .gitignore 

### ISSUE THREAD

**Issue by karolinakuzniewicz** [atom/atom#15836]
### Description

After creating new project, installing the dependencies and creating automatically the .gitignore file with **/node_modules** line, Atom doesn't ignore this. In Core Settings everything is set by default include **Exclude VCS Ignored Paths** (Checked/True). Unchecking/checking this option is not affecting on project's tree view. It seems like Atom isn't reading this local .gitignore file but in my global gitignore file I have the same settings.

### Versions

MacOS Sierra
Atom 1.21.0
git 2.14.2

**Comment by karolinakuzniewicz** [comment]
Reboot the system and Atom solved the problem.

**Comment by silasabbott** [comment]
@karolinakuzniewicz — to clarify, restarting the _computer_ solves this issue? I am having this problem as well, and Atom does not resolve the issue by quitting and reopening Atom. I have not tried rebooting my machine yet.

macOS High Sierra (10.13.2)
Atom 1.23.3
git 2.13.6

**Comment by karolinakuzniewicz** [comment]
@silasabbott are you sure that your project is using git? For me that solved my problems, I think that  Atom at that moment just couldn't load the changes

**Comment by silasabbott** [comment]
I am positive, I ran `git init` and have been using git through the Terminal alongside Atom.

Currently, all other files in my _.gitignore_ have been working just fine—any changes made are not tracked and do not reflect in the Unstaged Changes. However, one file that I added later still shows up in Unstaged Changes, and I can stage it as well. Atom won't let me commit the file though. Before I added the file to _.gitignore_, I did discard a change in the file (the first discard I've done in this project) and the "Undo Discard" button came up. It stayed after I added the file to _.gitignore_. The file I added does turn grey in the file browser, but otherwise it still acts like it was being tracked by git. I then decided to undo the discard and merge the conflicts to see if that was confusing Atom, but the file still acted the same. I then removed the file from my _.gitignore_, restarted Atom, added the file back in, and restarted Atom, but it still shows up when it's modified in Unstaged Changes. I have restarted Atom periodically throughout this process.

I just now modified the file while it was listed under _.gitignore_, and then checked `git status` to see if it showed as modified and unstaged—and YES. Git is not ignoring this file either, it showed as modified and unstaged. I must be doing something wrong, or git is doing something wrong, or this is for some reason the intended behavior.
